doc/overview/EDITING.md

```diff
@@ -1,125 +1,213 @@
+
+### What to do
+Often, these errors should be fixed by [asegdkt_segfile](#asegdkt_segfile) `<subject_dir>/mri/aparc.DKTatlas+aseg.deep.manedit.mgz`, but if that is not successful: 
+1. Open and edit  `<subject_dir>/mri/wm.mgz` and/or `<subject_dir>/mri/filled.mgz`.
+2. [Re-run FastSurfer](#general-process) to propagate the edits.
+
+The manual label 255 indicates a voxel should be included in the white matter and a voxel labeled 1 should not.
+
+See also: <https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/WhiteMatterEdits_freeview>
+
+## Pial surface placement
+
+### When to use
+Over- and/or under-segmentation of the cortical gray matter: voxels that should be gray matter are excluded, or those that should not are included.
+
+This explicitly 
+
+### What to do
+Often, these errors should be fixed in [asegdkt_segfile](#asegdkt_segfile) `<subject_dir>/mri/aparc.DKTatlas+aseg.deep.manedit.mgz`, but if that is not successful:
+1. Open end edit `<subject_dir>/mri/brain.finalsurfs.manedit.mgz` (overwriting values in `<subject_dir>/mri/brain.finalsurfs.mgz`).
+2. [Re-run FastSurfer](#general-process) to fix the pial surface. 
+3. 
+The manual label 255 indicates a voxel should be included in the gray matter and a voxel labeled 1 should not.
+
+See also: <https://surfer.nmr.mgh.harvard.edu/fswiki/Edits#brain.finalsurfs.mgz>
+
+## Side effects of edits
+
+Technically, all edits are changes to the processing pipeline and may cause systematic effects in the analysis.
+Computational edits (e.g. [Bias Field Correction](#bias-field-correction)) should be integrated for all cases of an analysis.
+Other effects are impossible to avoid (edit carefully), often very small and hard to account for. 
+It is recommended to analyze whether files with edits distort results in a specific direction, i.e. how do effect sizes change if manually edited cases are excluded?
+
+## Guide for experienced FreeSurfer users
+
+The following list enumerates editing options available in FreeSurfer and how to achieve similar edits in FastSurfer. 
+
+1. Skull stripping by editing `<subject_dir>/mri/brainmask.mgz` (delta to `brainmask.auto.mgz`) -> [asegdkt_segfile](#asegdkt_segfile)
+2. Talairach registration by editing `<subject_dir>/mri/transforms/talairach.xfm` (delta to `transforms/talairach.xfm`) -> [Talairach Registration](#talairach-registration)
+3. Brain extraction by editing `<subject_dir>/mri/brain.mgz` (delta to `brain.auto.mgz`) -> [asegdkt_segfile](#asegdkt_segfile)
+4. Subcortical segmentation: `<subject_dir>/mri/aseg.presurf.mgz` (delta to `aseg.presurf.auto.mgz`) -> [asegdkt_segfile](#asegdkt_segfile)
+5. CRS seeds: PONS, CC, LH, RH, watershed => seed files (`<subject_dir>/scripts/seed-(pons|cc|lh|rh|ws).crs.man.dat`) -> [asegdkt_segfile](#asegdkt_segfile)
+6. WM control points: `<subject_dir>/tmp/control.dat` -> [asegdkt_segfile](#asegdkt_segfile)
```

Comment:
hm, we could allow them to create control points which should be used by recon-surf when crating wm.mgz?

Reply:
I think we could, but there are some comments somewhere why we would not want to do that. I think I did not really understand those comment but just assumed you had decided against this on principle.

Reply:
Also, I am not really sure how these labels would propagate back into aseg.mgz and aparc.DKTatlas+aseg.mgz

Reply:
They would not, they are only used to increase intensity in thin WM regions to avoid loosing a gyrus or parts of a lobe (usually temporal lobe) due to low grey white contrast there. Let's discuss.